### DISCUSSION TITLE
Qual style guide vocês usam?

### DISCUSSION THREAD

**Discussion by danilovaz** [frontendbr/forum#129] [Perguntas e Respostas]
Eu encontrei alguns generators de style guide bem interessantes, como o [Hologram](http://trulia.github.io/hologram/) e o [mdcss](https://github.com/jonathantneal/mdcss).

No meu caso vi mais vantagens em utilizar o **mdcss** já que o **hologram** acaba me obrigando a ficar _refém_ do ruby. E meu objetivo no atual projeto que estou desenvolvendo, é buscar não amarrar o projeto.

Queria saber se alguém usa style guides? E se usa, quais generators utiliza e quais considerações tem?

Valeu! :smile: :

**Comment by felipefialho** [comment]
Pior que não uso nenhum, talvez esteja na hora de aderir algum, especialmente para projetos open source que envolvem mais gente. 

Gostei do mdcss.

**Comment by danilovaz** [comment]
@LFeh eu estou começando a usar agora também, vou testar o mdcss, me pareceu um aliado interessante pra ajudar a manter os padrões no projeto, principalmente pra quem está entrando no projeto.

**Comment by felipefialho** [comment]
Bom, vou começar um projeto novo aqui e aproveitar pra testar esse mdcss ;)

**Comment by danilovaz** [comment]
@LFeh legal, depois que tiver uma opinião sobre o uso me diz o que achou, ainda tô vendo os benefícios de se usar

**Comment by diessica** [comment]
Conheço esses também:
- http://documentcss.com/
- http://documentjs.com/

**Comment by mksanti** [comment]
Cara eu tava procurando isso um dia deste! muito bom esse mdcss.

**Comment by willianjusten** [comment]
A HUGE tem um styleguide, acho ele bem maneirinho também.

http://hugeinc.github.io/styleguide/

**Comment by danilovaz** [comment]
@willianjusten cool, funciona com stylus? vi vários que só aceitam sass ou css puro

**Comment by willianjusten** [comment]
"CSS Styles

This projects supports SASS, LESS and Stylus. Choose one and use the file with the matching extension."

=D

**Comment by danilovaz** [comment]
@willianjusten my bad, ainda não tinha visto a página de modules :smile:

**Comment by willianjusten** [comment]
Relax, foi só para confirmar que tem sim =D

**Comment by willianjusten** [comment]
Eu vi esse ainda pouco, achei legal hein.

http://devbridge.github.io/Styleguide/

**Comment by felipefialho** [comment]
@willianjusten 

Estou tendo alguns problemas com o MDCSS.

Gostei muito desse último sabe se funciona com Stylus?

**Comment by danilovaz** [comment]
@LFeh ainda não tem -> [support to stylus](https://github.com/devbridge/Styleguide/issues/119) mas ele deu a letra de criar um arquivo .scss enquanto não lança suporte pra stylus. Tive alguns problemas com o mdcss tb, começou bem mas depois quebrou tudo

**Comment by felipefialho** [comment]
Acabei de ver o @danilovaz abrindo issue lá. https://github.com/devbridge/Styleguide/issues/119

Com relação ao MDCSS, estou com esse problema https://github.com/jonathantneal/mdcss/issues/9, mas não consigo uma solução plausível.

**Comment by grvcoelho** [comment]
Sem querer vender o peixe, mas: https://github.com/grvcoelho/css

**Comment by danilovaz** [comment]
@grvcoelho na verdade, seria algo mais interativo [assim](https://devbridge.github.io/Styleguide/) mas style guides como você montou aí também são uma boa...acho uma boa ter os dois

**Comment by aleUX** [comment]
Esse site aqui tem uma porrada de referencias e informações sobre style guide http://styleguides.io/
Uma parte dele cita alguns generators

**Comment by felipefialho** [comment]
Boa @alexandrehappybiz \o

**Comment by filipelinhares** [comment]
Eu fiz um [style guide generator](https://github.com/filipelinhares/toth) porque os que eu conhecia eram muito chato de ficar configurando. Depois de uma semana apareceu o [mdcss](https://github.com/jonathantneal/mdcss) que é muito bom e simples tb.

_ps: Tenho que melhorar algumas coisas no meu mas da pra usar._
